examples/remote-git-sub/02-placement.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: Placement
+metadata:
+  name: nginx-demo-placement
+  namespace: default
+spec:
+  numberOfClusters: 1
```

```suggestion
  numberOfClusters: 10
```

lets just do 10, one seems too little

---

10 sounds good, updated it now